packages/fast-components-react-base/src/radio/README.md

```diff
@@ -0,0 +1,8 @@
+# Radio
+An element that can be turned on and off. *Radio* buttons are almost always listed together in groups.
+
+## Usage
+Only one *radio* within a *radio* group may be selected at a time. Limit the *radio* label to a single line.
+
+## Style guidance
+Should not put two radio groups next to each other. When two radio groups are right next to each other, it’s difficult to determine which radios belong to which group. Use group labels to separate them.
```

Comment:
"Do not put two radio groups next to each other."

Reply:
Also style guidance should only be in MSFT. So move this change there

Reply:
removed style guide from here